base/bucket_gocb.go

```diff
@@ -147,6 +152,72 @@ func (bucket CouchbaseBucketGoCB) GetBucketCredentials() (username, password str
 	return bucket.spec.BucketName, password
 }
 
+func (bucket CouchbaseBucketGoCB) GetMetadataPurgeInterval() (int, error) {
```

Comment:
Did you check if gocb has an API to get this?  Do you think it's worth filing a feature request?